app/controllers/api/v1/keywords_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class KeywordsController < ApplicationController
+      include ::Pagy::Backend
+      include API::V1::Pagy::JSONAPIConcern
+
+      def index
+        pagy, keywords_list = pagy(keywords)
+
+        if keywords.any?
+          render json: KeywordSerializer.new(keywords_list, pagy_options(pagy)).serializable_hash, status: :ok
```

Comment:
Does it render `200` status code without specifying `status: :ok`?

Reply:
Yes! :) I'll remove the xtra code then :) Thanks!

Reply:
Done ;-)